guides/GITHUB.md

```diff
@@ -0,0 +1,343 @@
+If you like working with configuration files and workflows, `sourcecred-action`
+might be for you. If you aren't a programmer, you might be most comfortable starting
+with `docs`.
+
+
+### Is my help needed in a software project when I'm not a programmer?
+
+Yes! Any substantial software project is multi-disciplinary. This one included. In fact, calling SourceCred a software project doesn't do justice to how much work besides programming goes into it.
+
+
+### Why and how should I get involved here specifically?
+
+It's easy to think:
+
+> GitHub seems like a platform for programmers. I'm not a programmer, why would anyone want my help?
+
+But here's the thing (cue Monk)! When you look deeper at a lot of discussions
+that are had around GitHub repositories, while some of them might require code 
+expertise, a large portion do not. Even when the changes are specifically to
+suggest changes to code, it's usually the responsibility of the individual
+asking for feedback to explain their changes in human friendly terms. For
+example, a developer might say something like:
+
+> This (review request) is going to add a plugin to support reddit as a place to get cred.
+
+Even if you know or understand nothing about the code, you can contribute by
+asking questions. What are the actions that can generate it? How has this
+new integration been tested? By asking these questions, you can start to
+participate in conversations that you might have felt that you couldn't
+contribute to. As you do this more, you'll learn and over time learn more
+about the actual code or documentation base. So in summary, don't
+be afraid to just jump in. You have meaningful questions to ask, or comments
+to make. You are the only one judging yourself for not
+having some perceived level of expertise.
+
+
+## What mindset should I have?
+
+Learning new things is challenging. It's important to realize that since using
+GitHub and git is entirely new to you, it's going to be challenging. You'll 
+likely start out and not entirely be sure about what the commands you are
+copy pasting are doing. There are a few things to remember here.
+
+### Inexperienced != Incapable
+
+Every current user of GitHub, or any technology for that matter, was new at 
+some point. Think of someone that you consider proficient or expert, and 
+realize that they at some point were in the same shoes as you. The
+fact that you are new to a technology says nothing about what you
+are capable of, of your innate abilities.
+
+### It's okay to ask for help
+
+What does it mean to ask for help? You might have questions, feedback, or
+a general comment, or just want to talk to someone. Asking for help can be daunting, 
+but guess what? The SourceCred community is open to it. Once you've
+sprinted over the mental hurdle that it's okay to ask, you'll very likely
+realize that engaging in the first place is a great way to learn, and quite a bit
+of fun. 
+
+
+### I will grow
```

Comment:
I'm skeptical that we should have these mindset sections at the front of the doc. It feels like, _too_ hand-holdey, personally I would rather get to content. however, i recommend I'm not the target audience for the doc -- curious for feedback from e.g. LB